Readme.md

```diff
@@ -1 +1,43 @@
 # Chaqer
+Program to check if a person is in the image or not
+
+Before running the programs in the order mentioned below export your Ocp-Apim-Subscription-Key you got from Microsoft as an environment variable
+
+# Create Person Group
```

Comment:
This should be a secondary level header.

```
## Create Person Group
```

Also show an example of each secondary level header you wrote about.